pyintegration/deephaven2/agg.py

```diff
@@ -0,0 +1,256 @@
+#
+#  Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+#
+from __future__ import annotations
+from typing import List, Any
+
+import jpy
+
+_JAggregation = jpy.get_type("io.deephaven.api.agg.Aggregation")
+
+
+class Aggregation:
+    """ A Aggregation object represents an aggregation operation.
+
+    Note: It should not be instantiated directly by user code but rather through the static methods on the class.
+    """
+
+    def __init__(self, j_aggregation):
+        self._j_aggregation = j_aggregation
+
+    @property
+    def j_agg(self):
+        return self._j_aggregation
+
+    @staticmethod
```

Comment:
I expected these to be module-level functions.  Having them as static functions seems to add complexity without obvious benefit in this design.  With these as static methods, users can not import them individually, which may lead to nastier code in some cases.  What am I missing?
https://stackoverflow.com/questions/12603541/import-only-a-class-static-method